custom/icds_reports/ucr/data_sources/usage_forms.json

```diff
@@ -57,6 +57,20 @@
           "name": "user_location_id"
         }
       },
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "type": "expression",
+        "column_id": "supervisor_id",
+        "type": "expression",
+        "expression":{
+          "type": "location_parent_id",
```

Comment:
Used this as there is only one instance. Assuming the usage_forms are submitted by AWC, this is correct